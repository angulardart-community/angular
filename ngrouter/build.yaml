targets:
  $default:
    builders:
      ngdart:
        generate_for:
          exclude: ["test/url_test.dart"]
