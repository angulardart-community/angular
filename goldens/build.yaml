targets:
  $default:
    builders:
      build_web_compilers:entrypoint:
        generate_for:
          - test/**.dart
        options:
          dart2js_args:
            - -O1
            - --no-source-maps
